docs/home/howto/CLUSTER.md

```diff
@@ -649,3 +649,109 @@ You can read more [here](https://www.confluent.io/blog/how-choose-number-topics-
 ## RabbitMQ installation and setting
 
 To install RabbitMQ in single mode just add rabbitmq role to your data.yaml for your server and in general roles section. All configuration on RabbitMQ - e.g. user other than guest creation should be performed manually.
+
+## Howto use azure scalability set feature
+
+In your cluster yaml config declare several (if required) object of kind `infrastructure/availability-set` like
+example below, change the `name` field as you wish.
+
+```yaml
+---
+kind: infrastructure/availability-set
+name: kube-node-availability-set
+provider: azure
+specification:
+  name: kube-node-availability-set
+  platform_fault_domain_count: 2
+  platform_update_domain_count: 5
+  managed: 'true'
+```
+
+Then your infrastructure type `infrastructure/virtual-machine` with the name you want to enable (below is
+kubernetes-node-machine) reference the entry, for example:
+
+```yaml
+kind: infrastructure/virtual-machine
+name: kubernetes-node-machine
+provider: azure
+specification:
+  size: Standard_DS3_V2
+# This tell to template the vm terraform to insert the id of the availability-set
+  availability_set: kube-node-availability-set
+title: Virtual Machine Infra
+```
+
+Then in the corresponding `components` section of the kind `kind: epiphany-cluster` set it as well.
+
+```yaml
+  components:
+    kafka:
+      count: 0
+    kubernetes_master:
+      count: 1
+    kubernetes_node:
+# This line tell we generate the availability-set terraform template
+      availability_set: kube-node-availability-set
```

Comment:
@sunshine69 on which components did you test such configuration?

Reply:
The current prodeastus and devdr10, devcluster one I did create for all components type and it works for all. Kubernetes master, nodes, kfafka, postgres etc, each of them has its own availability set.

Reply:
Ok, one more thing, isn't that line repetition with line 680 in `kind: infrastructure/virtual-machine` section?

Reply:
Not sure what you mean by repetition, as of now we need to define in several places like the sample config.

I am trying to see what needs to be done to remove the settings in the `kind: infrastructure/virtual-machine` without adding too much complexity.